### DISCUSSION TITLE
Move the Find Dialog

### DISCUSSION THREAD

**Discussion by ghost** [VSCodium/vscodium#2134] [Q&A]
Is it possible to move the find dialog to one of the side panels? I'd rather have it live on a side panel rather than drop down (and push all text down) from the top of a document.

**Comment by dedWork** [comment]
Howdy, 

Unfortunately I don't believe this is currently possible.
Here's an older discussion about it in VSCode; https://github.com/Microsoft/vscode/issues/26498

**Comment by ghost** [comment]
Skimming that thread it feels like a Monkey Paw situation. Hey this dialog causes a lot of issues so rather than allow it to be placed like all the other elements lets just make it push the text out of the way...le sigh, always something heh. Anyhow thanks for the heads up.

**Comment by Jeong-ilKajokaya** [comment]
Hi! Currently, VSCodium (and upstream VSCode) does **not support moving the Find/Replace dialog to a side panel**. The find widget is designed to drop down from the top of the editor and is not a standalone panel or view.

### Alternatives you might consider:

1. **Use the Search Sidebar**

   * Press `Ctrl + Shift + F` (or `Cmd + Shift + F` on macOS) to open the **Search** panel in the sidebar.
   * It supports searching across files, highlighting matches, and replacing across files.
   * This doesn’t replace the inline find in a single file, but it is a persistent sidebar search.

2. **Extensions**

   * Some extensions might offer enhanced search UIs or floating search panels, but as of now, moving the native find widget to a side panel is not supported.

3. **Feature Requests**

   * You can submit a feature request to the upstream VSCode repo if this is an important workflow improvement.

**Summary:**

* Native inline find dialog: cannot move to sidebar.
* Use Search sidebar for persistent side-panel search.
* Check for extensions or submit a feature request for future support.

**Reply by ghost** [reply]
Ug more useless LLM slop replies...

**Reply by Jeong-ilKajokaya** [reply]
What's wrong with LLMs? 
Also, did it answer your question? I'm just trying to help, bro. No need to yell.

**Reply by ghost** [reply]
It didn't answer anything, it rather literally stated everything already known as outlined by the problem. LLM's are useless. No one was yelling, just sick of LLM slop that says a lot of nothing...like that reply. LLM speak is like listening to a politician ramble on about everything that avoids the topic or giving any answer to the question...

**Reply by Jeong-ilKajokaya** [reply]
Useless is a _**strong**_ adjective for LLMs. Others have found help with my LLM-assisted answers. 
I am sorry that my answer didn't help you. I didn't mean to lie or mislead you. I was just offering my help to others.

Next time, I will get the LLM to write a better answer that with solve the next person's problem. Cool? 

I am new, so I am still learning how to give better answers. So again, my bad, I am working on it. My future will be better I promise bro 🙏🏿

![giphy](https://media1.giphy.com/media/v1.Y2lkPTc5MGI3NjExanBlb3VpdHdheWg4eDFwM2g3Y3d2MjB5YTI0bnM3ZTBjajd0ZGk5diZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/6pxG2dThniE5G/giphy.gif)

**Reply by ghost** [reply]
Useless is the correct term. If you find them useful good for you, I've yet to find any model on any hardware that isn't -useless. They lie, make things up, can't given you basic yes or no answers if they answer at all. They stall and dodge the question most the time.

There should be no "next time" for an LLM to write an answer until said LLM can comprehend it has nothing to add, or shocker can simply say "no" rather than blurt out a lot of crap no one asked for that isn't on topic. "My bad" sounds like the pot head co-pilot model...

**Reply by Jeong-ilKajokaya** [reply]
I don't like your tone. Why are you hating on AI for? Just because you had a few bad experiences with AI doesn't mean they are useless. How would feel if someone called you useless just because they had a few bad interactions with you?

![giphy](https://media.giphy.com/media/c967fyU75FO6sN8aqZ/giphy.gif?cid=790b7611ee382cd21fef02378618a59061b4127fd8bfd4e6&ep=v1_user_favorites&rid=giphy.gif&ct=g)

I’m not saying AI is perfect — it still makes mistakes, lacks context at times, and can’t fully replace human expertise or creativity. But pretending that AI “can’t get real work done” is just not true.

AI can deliver useful, high-quality work when used correctly: it can speed up research, draft content, suggest solutions, automate repetitive tasks, generate ideas, and even help debug or write code. The key is knowing how to guide it, review the output, and apply your own judgment on top of it.

It shouldn’t be viewed as a replacement for humans, but as a powerful tool that enhances what humans can do — especially for brainstorming, productivity, and accelerating results.

How do I know you are not using AI to hate on AI? That would be ironic 
<br>
![giphy](https://media1.giphy.com/media/v1.Y2lkPTc5MGI3NjExZmd3ZTJ5OXBuZ3Y4YW9zdjkycTZsZnl2dGg0ZGpyZXM4M3luMWg0OCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/BmX38GoChnxRe/giphy.gif)

**Reply by daiyam** [reply]
@aaronthefullstackengineer Please be mindful that not every one like LLM. And your answers are generally over the top.

In this case, your answer wasn't helpful.

And @nPHYN1T3 hasn't commented on you but your use of IA. So please avoid any personal attack.

Thx

**Reply by Jeong-ilKajokaya** [reply]
My bad bro, I will chill. I swear @daiyam 

![giphy](https://media.giphy.com/media/v1.Y2lkPWVjZjA1ZTQ3YTdkemFuZmpkYWU0eXl0ejN2MXBleDJyMWl4bmp3OWV1djhzYWg1dyZlcD12MV9naWZzX3NlYXJjaCZjdD1n/XkNv6S9ardASk/giphy.gif)

I get fucking furious when people hate on AI for no real reason. Most of them can’t even explain why they’re against it — they just parrot whatever nonsense they heard online like brain-dead NPCs. AI is one of the most powerful breakthroughs of our era. Yeah, the concept has existed since the 50s, but only now are we seeing a real revolution that’s changing the world.

And yet, people are ready to throw it all away because some loud, clueless voices told them to be scared. They don’t think. They don’t research. They just follow the herd.

It pisses me off. What the hell has AI ever done to you? Why are you so terrified of progress?

If you’re going to talk shit about AI, at least have a brain cell to back it up. @nPHYN1T3 

![giphy](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExNWYxNm9pZHhwMmJrdWFuajBzY3NhN3NjMjN2MWI4c256eGRkeWQ3YiZlcD12MV9naWZzX3NlYXJjaCZjdD1n/gkWlSZvUNmi20adIre/giphy.gif)

**Reply by ghost** [reply]
If this makes you so angry you should write a blog not rant on question posts. I gave clear reasons why I do not like Ai/LLM and I never said it will always be useless but for now, it is. An absurdly massive waste of power and compute to spit out made up "facts" that derail the whole purpose of being a "crystal ball" of answers. Much like as demonstrated with your answer it was "a lot about nothing." The answer to this post (for now) is one word, "no." Yet your response might have been tell me to use a different IDE, stand on my head or anything else that is not about moving the find bar. Telling me to use the bloated search side bar? I didn't ask about the side bar. Mentioning "Maybe, possibly there MIGHT be some addons..." How about TELL ME of these magic addons. This is a "classic" LLM farse where it presents a "you should look into that" scenario as an answer. As for my brain cells perhaps your reading comprehension could use a tune up along with your memes.

**Reply by Jeong-ilKajokaya** [reply]
> An absurdly massive waste of power and compute to spit out made up "facts" that derail the whole purpose of being a "crystal ball" of answers

WASTE OF POWER AND COMPUTE 😱😱. Watch your mouth. AI lives matter. 
![giphy](https://media.giphy.com/media/v1.Y2lkPWVjZjA1ZTQ3b2hkdDdvbjF5ZXR0NGY3cHczZTQ5YWd6NHM0M2gxNnlzNnhkbTFyMCZlcD12MV9naWZzX3NlYXJjaCZjdD1n/rW2TXSfejR4Ox4ZVRg/giphy.gif)


> Yet your response might have been tell me to use a different IDE, stand on my head or anything else that is not about moving the find bar.

Nah, nah, nah. You’re twisting things just to push this tired narrative that AI is trash. Be serious. By that logic, every search engine on the planet is garbage too, just because it didn’t hand you the perfect result on the first try. That’s not a real argument — that’s just reaching.



> Telling me to use the bloated search side bar? I didn't ask about the side bar. Mentioning "Maybe, possibly there MIGHT be some addons..."

See, that’s on you. I was giving suggestions, and now you want to get mad at me and the AI because we can’t read your mind. That’s not our fault. If you don’t have the clarity or intelligence to craft a proper question that actually communicates your thoughts to the people trying to help, that’s on you — not us.



> As for my brain cells perhaps your reading comprehension could use a tune up along with your memes.

Excuse me? 🤨🤬 My memes are funny and relatable.
![giphy](https://media.giphy.com/media/v1.Y2lkPWVjZjA1ZTQ3ZXBidXlucWR0N3pzcHFkbWp0ZTE0aTh2a2U3ejk4NDR3YmFhbzM5biZlcD12MV9naWZzX3JlbGF0ZWQmY3Q9Zw/z75OjWknCTI6k/giphy.gif)

**Reply by daiyam** [reply]
> My bad bro, I will chill. I swear @daiyam
> 
> 
> If you’re going to talk shit about AI, at least have a brain cell to back it up. @nPHYN1T3

Start: good
End: immediate fail

I've said: no personal attack. It's a red card. Next time, it's a week ban.
